### DISCUSSION TITLE
Merging PR in vercel/async-sema repo

### DISCUSSION THREAD

**Discussion by jdalrymple** [vercel/community#4738] [Help]
### Summary

Hello! 

Any chance we could get this PR merged? https://github.com/vercel/async-sema/pull/51. This repo could use some love haha

### Steps to Reproduce

See issue: https://github.com/vercel/async-sema/issues/48 (to ensure the discussion happens in the same place)

**Comment by trevor-vaughan** [comment]
@amyegan Is async-sema still alive?
